index.html

```diff
@@ -0,0 +1,260 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+    <title>my Projects</title>
+  </head>
+  <body>
+    <h1 id="first">My Recent</h1>
+    <h1 id="sec">Works</h1>
+    <hr />
+    <section id="project-cards">
+      <div class="card">
+        <div class="img_container"></div>
+        <div class="bottom">
+          <div class="intro">
+            <h2>Mult-Post Stories</h2>
+            <h2>Grain+Glory</h2>
+          </div>
+          <div class="btn">
+            <button type="button" class="ruby">Ruby on rails</button>
+            <button type="button" class="css">css</button>
+            <button type="button" class="js">Javascript</button>
+            <button type="button" id="html">html</button>
+          </div>
+          <button type="button" class="btn-project">See Project</button>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="img_container"></div>
```

Comment:
- [ ] It is advisable to wrap the cards in a ul and each card should be a li element instead . See the example below.
```
<ul>
  <li>
  <!--card data-->
  </li>
  
  <li>
  <!--card data-->
  </li>
  
  <li>
  <!--card data-->
  </li>

</ul>
```